zipline-kotlin-plugin/src/main/kotlin/app/cash/zipline/kotlin/AdapterGenerator.kt

```diff
@@ -806,6 +840,11 @@ internal class AdapterGenerator(
       )
     }
 
+    // if this function is part of a property, box it
```

Comment:
@swankjesse do you remember if this is necessary?

Reply:
Already done. You can remove this comment.